client/knowledge_base_search/src/containers/QuestionBarContainer/QuestionBarContainer.jsx

```diff
@@ -63,7 +63,7 @@ class QuestionBarContainer extends Component {
                 })
               }
               type="button"
-              disabled={isFetchingResults}
+              disabled={isFetchingResults || query.question === currentQuery}
```

Comment:
since this is used right above, probably should move this into a helper of some sort